api/types/types.go

```diff
@@ -286,6 +286,7 @@ type ContainerJSON struct {
 // NetworkSettings exposes the network settings in the api
 type NetworkSettings struct {
 	NetworkSettingsBase
+	DefaultNetworkSettings
```

Comment:
As a note before people freak out: this is embedded, so there won't be any `DefaultNetworkSettings` key.